src/org/thoughtcrime/securesms/BaseConversationItem.java

```diff
@@ -103,6 +106,9 @@ protected class ClickListener implements View.OnClickListener {
 
     public void onClick(View v) {
       if (!shouldInterceptClicks(messageRecord) && parent != null) {
+        if (batchSelected.isEmpty() && ((AccessibilityManager) context.getSystemService(Context.ACCESSIBILITY_SERVICE)).isTouchExplorationEnabled()) {
```

Comment:
i would create a `Util.isTouchExplorationEnabled()` or so and do the check with a try/catch